mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -51,19 +51,18 @@ private[feature] trait QuantileDiscretizerBase extends Params
   def getNumBuckets: Int = getOrDefault(numBuckets)
 
   /**
-   * Relative error (see approxQuantile documentation for description).  Must be >= 0.
-   * By default, the relative error will be in the range [0.001, 0.01]. The exact value
-   * is computed based on the number of buckets, decreasing as the number of buckets
-   * increases. Explicitly setting the relative error will override this default calculation.
+   * Relative error (@see <a href="https://github.com/apache/spark/blob/13ce10e95401b21fa40ca0bb27ebf9a0bfffe70f/sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala#L66">approxQuantile</a> documentation for description).
```

Comment:
You can link to a method in Scaladoc, and the generated docs will have the correct link. An `@see` link will generate a separate "see also" section, so it may be better to in-line it here, like this:
```
* Relative error (see documentation for 
* [[org.apache.spark.sql.DataFrameStatFunctions.approxQuantile approxQuantile]] for description)
```